Modules/Server/Sources/PocketCastsServer/Public/Refresh/MainServerHandler.swift

```diff
@@ -229,8 +229,8 @@ public class MainServerHandler {
             return nil
         }
 
-        for podcast in podcasts { // ensure podcasts have up to date latest episode uuids
-            ServerPodcastManager.shared.updateLatestEpisodeInfo(podcast: podcast, setDefaults: false)
+        for (idx, podcast) in podcasts.enumerated() { // ensure podcasts have up to date latest episode uuids
+            ServerPodcastManager.shared.updateLatestEpisodeInfo(podcast: podcast, setDefaults: false, cache: idx == podcasts.endIndex)
```

Comment:
We don't need to update the cache until the last podcast is saved by default the cache is updated on each save, triggering a DB fetch each time.